sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/catalog/CosmosCatalogNotFoundException.scala

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.spark.catalog
+
+class CosmosCatalogNotFoundException (msg: String) extends CosmosCatalogException(msg)
```

Comment:
why public - same for other new classes

Reply:
Updated to use private[spark], same for all other new classes